pandas-stubs/core/arrays/interval.pyi

```diff
@@ -15,10 +18,14 @@ from pandas._libs.interval import (
 from pandas._typing import (
     Axis,
     Scalar,
+    ScalarIndexer,
+    SequenceIndexer,
     TakeIndexer,
     np_ndarray_bool,
 )
 
+IntervalOrNA: TypeAlias = Interval | float
```

Comment:
Do we need to keep this in this file or should we move it to _typing?

Reply:
probably `_IntervalOrNA`

Reply:
I want to be consistent with pandas for now.  This statement works:  `from pandas.core.arrays.interval import IntervalOrNA`

Reply:
Is this a shift in policy (from public == what is documented on the web) to more how the python community/type checkers defines public/private? That is fine with me, just want to make sure if that is the case.

Reply:
> Is this a shift in policy (from public == what is documented on the web) to more how the python community/type checkers defines public/private? That is fine with me, just want to make sure if that is the case.

Not really.  I just want to be consistent right now with the types in `pandas`.  I think we need to change some of types likes these that are in `pandas` to be private (precede with underscores), which is a separate effort.